cmd/operator/main.go

```diff
@@ -271,6 +272,25 @@ func run() int {
 		return 1
 	}
 
+	// Check if we can read the storage classs
+
+	canReadStorageClass, err := checkPrerequisites(
+		ctx,
+		logger,
+		cc,
+		allowedNamespaces,
+		map[string][]string{
+			storageclassv1.GroupName: {"get"},
+		},
+		storageclassv1.SchemeGroupVersion.String(),
+		storageclassv1.GroupName,
+	)
```

Comment:
This looks like an overkill, the arguments are pretty small to occupy one line per arg. Is this the way, they're doing it everywhere else in the code? Better way would be to club smaller items in single line and break the line where it gets too long.

Reply:
Yes, that is how they are doing it when calling checkPrequisites everywhere in the file

Reply:
https://github.com/infracloudio/prometheus-operator/blob/b0aa730488f093be9fa39409d49d9cd8a67edc77/cmd/operator/main.go#L274-L289